core/src/main/java/com/redhat/lightblue/client/request/data/DataDeleteRequest.java

```diff
@@ -31,6 +31,7 @@ public JsonNode getBodyJson() {
         if (queryExpression != null) {
             node.set("query", queryExpression.toJson());
         }
+        appendExecutionToJson(node);
```

Comment:
Range is not used for delete requests, so can't be set in AbstractLightblueDataRequest (common base).